.pre-commit-config.yaml

```
@@ -34,6 +34,13 @@ repos:
       - id: codespell
         additional_dependencies: [tomli]
 
+  - repo: https://github.com/PyCQA/docformatter
+    rev: v1.7.5
+    hooks:
+      - id: docformatter
+        additional_dependencies: [tomli]
+        args: ["--in-place"]
```

Looks like there are the necessary options:
```suggestion
        args: ["--in-place", "--pre-summary-newline", "--make-summary-multi-line"]
```
This is to keep the changes of this PR to the docs at a minimum.